src/main/java/com/job/BadCode.java

```diff
@@ -0,0 +1,44 @@
+package com.job;
+
+/**
+ * Created by prashil on 7/27/17.
+ */
+public class Calculation {
+
+
+  public static void main(String[] args) {
+    float a = 8.2f;
+    float b = 30.13f;
+
+    float result = ((21f / 24f) * 100f * a) + (100f * a) + ((3f / 24f) * 100f * b);
+    System.out.println("3 - 6 : " + result);
```

Comment:
Okay cool sounds good. Just change that float and then I can approve you request.